lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.BoilerHotWater.rb

```diff
@@ -0,0 +1,88 @@
+class ASHRAE901PRM < Standard
+  # @!group BoilerHotWater
+
+  # find search criteria
+  #
+  # @return [Hash] used for standards_lookup_table(model)
+  def boiler_hot_water_find_search_criteria(boiler_hot_water)
+    # Define the criteria to find the boiler properties
+    # in the hvac standards data set.
+    search_criteria = {}
+    search_criteria['template'] = template
+
+    # Get fuel type
+    search_criteria['fuel_type'] = 'Gas'
+
+    # Get the fluid type
+    search_criteria['fluid_type'] = 'Hot Water'
+
+    return search_criteria
+  end
+
+  # Finds lookup object in standards and return minimum thermal efficiency
+  #
+  # @return [Double] minimum thermal efficiency
+  def boiler_hot_water_standard_minimum_thermal_efficiency(boiler_hot_water, rename = false)
```

Comment:
The methods in this file are missing YARD comments for the parameters.

Reply:
Added